modules/cloudbuild/main.tf

```diff
@@ -149,10 +149,10 @@ resource "google_project_iam_member" "org_admins_source_repo_admin" {
 }
 
 /***********************************************
- Cloud Build - Master branch triggers
+ Cloud Build - Main branch triggers
  ***********************************************/
 
-resource "google_cloudbuild_trigger" "master_trigger" {
+resource "google_cloudbuild_trigger" "main_trigger" {
```

Comment:
I'm not sure this is worth the state migration - @bharathkkb, thoughts?

Reply:
@morgante since this a breaking change anyway we could provide a small upgrade guide with a state mv command to be inline with https://developers.google.com/style/inclusive-documentation#replace-or-write-around-non-inclusive-terms

Reply:
Yeah I don't mind either approach.